lib/ToggleButton/ToggleButtonAccessibilitySupport.js

```diff
@@ -0,0 +1,28 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name ToggleButtonAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: 'value'}
+	],
+
+	/**
+	* @private
+	*/
+	updateAccessibilityAttributes: kind.inherit(function (sup) {
+		return function (was, is, prop) {
+			var enabled = !this.accessibilityDisabled;
+			sup.apply(this, arguments);
+			this.setAttribute('aria-pressed', enabled ? String(this.value) : null);
+			this.setAttribute('aria-labelledby', enabled ? this.getId() : null);
```

Comment:
@ryanjduffy One thing I am concerning is moonstone should consider all platform (including PC ChromVox)? Enyo core definitely should but moonstone? webOS TV, Android talkback and PC Chromvox support ARIA all differently. Specially, chromVox works much less and different from iOS, android and webOS TV Blink. Well...the best is we can support all platform but it makes our code complicated in some cases. Good example is aria-hidden with aria-labelledby. My opinion is we will consider webOS and chromVOX as much as possible, but if moonstone code goes too much complicated then I'd focus on TV case. Anyway, we can do both for togglebutton with bongsub's suggestion.

For making `accessibilityLabelledBy` by wrapping `aria-labelledby`, I feel like not to do right now, we need take more time to check many cases. `aria-labelledby' works mutually exclusive with `aria-label` and if user uses `accessibilityLabel` and `accessibilityLabelledBy` together then we need to make policy all cases. For short term, rather to open API, i'd add `aria-labelledby` inside of component code whenever it needs

CC @graynorton